src/config.c

```diff
@@ -0,0 +1,66 @@
+#include "libblastrampoline_internal.h"
+
+lbt_config_t lbt_config;
+
+void init_config() {
+    lbt_config.loaded_libs = (lbt_library_info_t **)malloc(sizeof(lbt_library_info_t *)*(MAX_TRACKED_LIBS + 1));
+    memset(lbt_config.loaded_libs, 0, sizeof(lbt_library_info_t*)*(MAX_TRACKED_LIBS + 1));
+    lbt_config.build_flags = 0x00000000;
+}
+
+JL_DLLEXPORT const lbt_config_t * lbt_get_config() {
```

Comment:
I guess you took this from Julia source code, but for consistency shouldn't the macro be called `LBT_DLLEXPORT`?